config/prow/jobs/custom/prow-autobump.yaml

```diff
@@ -0,0 +1,74 @@
+periodics:
+- cron: "15 15-23 * * 1-5"  # Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
+  name: ci-oss-test-infra-autobump-knative-prow
+  cluster: test-infra-trusted
+  decorate: true
+  extra_refs:
+  - org: GoogleCloudPlatform
+    repo: oss-test-infra
+    base_ref: master
+  annotations:
+    testgrid-dashboards: googleoss-test-infra
```

Comment:
Probably should go under a `prow` dashboard or `autobump` or something.. there is a `utilities` dashboard, but don't know if that is the right place.